### ISSUE TITLE
Send to SD via Cloud

### ISSUE THREAD

**Issue by Lesani** [bambulab/BambuStudio#1783]
**Is your feature request related to a problem? Please describe.**
My printer lives on a different LAN than the PC I work from. I can print directly via the cloud service, but when I prepare a file in advance, and want to send it to the printer while the printer is currently printing, I cannot do that because the "Send" function in contrast to the "Print" function, only wants to communicate on the same network and does not use the cloud service.

![image](https://user-images.githubusercontent.com/5182104/237045263-06999fd8-6a33-44e3-bd9f-fb7543d4b18f.png)

**Describe the solution you'd like**
Enable sending a gcode file via the cloud service, same as direct printing

**Describe alternatives you've considered**
I have no alternative, I cannot get on the same LAN (large industrial company, not home user)

**Additional context**

**Comment by walterwongbbl** [comment]
Hi.
Printing files can also be copied to the printer using an SD card.
Does this meet your needs?

**Comment by Lesani** [comment]
No, because it is additional ways between the printer, maintaining the SD card and fetching it separately. You already have a system in place to send the GCode to the printer via the cloud. Please extend it to be also able to send a file, without directly starting a print

**Comment by walterwongbbl** [comment]
Hi.
I have collected your suggestions. We will discuss this feature in the next requirements discussion

**Comment by michi1g** [comment]
+1 for this feature

**Comment by morganvenable** [comment]
+1.  Ridiculous that a file sent to the printer can't be stored locally for later reference.  Makes storing/selecting prints a very unnecessary hassle.

**Comment by dopesong** [comment]
+1

**Comment by antonmorch** [comment]
+1. I faced the problem for the first time today, and wondered why it wasn't possible, since I didn't enable LAN-only mode. 
Turns out it isn't possible at all to send via cloud.

**Comment by pleite** [comment]
+1 It's a mess to have to bring a PC to the printer lan, as it's another building, because we can't just load a sliced model. For example to place adaptive layers or combine parts without making a published model for handy, or just put some gcode without geometry.

**Comment by Kucharskov** [comment]
+1 I solved it by creating Tailscale VPN between location of printer and my home to be able to send files as "in local network" even remotely
